### PR TITLE
added readme cluster

### PR DESCRIPTION
**Pull request by cguerain**
Readme.md for cluster part added.

### CODE REVIEW

File: apps/cluster/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Code Example
+
+
+```hcl-terraform
+module "consul-cluster" {
+  source = "git@github.com:albumprinter/eops_tf_modules.git//apps/cluster?ref={TAG_VERSION}" # for eg. {TAG_VERSION} = v0.7.0
+    tags_team = "eops" //team name
+    environment = "dev-nonprod-prod" 
+    app_name = "eops-consul" //name of the application for cluster
+    app_elb_name = "eops-consul_ELB" //name of the loadbalancer
```
Comment (zorancco) [inline apps/cluster/README.md]: Let's create a follow up case to streamline the names, also we should add in a parameter for custom json chef provisioner!

**Comment by zorancco** [comment]
Can you update the changelog and bump the version number there first?

**Comment by cguerain** [comment]
Just did with the adjusted pull request

File: apps/cluster/README.md (hunk 1 above)
Comment (cguerain) [inline apps/cluster/README.md]: Case will be created.

**Comment by cguerain** [comment]
version  #0.7.1

**Pull request merged**